### ISSUE TITLE
Which combination of dependency versions is supposed to work?

### ISSUE THREAD

**Issue by graphicore** [googlei18n/glyphs2ufo#8]
I was just trying out the `glyphs2ufo` command and got the following error:

```
Traceback (most recent call last):
  File "/usr/local/bin/glyphs2ufo", line 9, in <module>
    load_entry_point('glyphs2ufo==1.0', 'console_scripts', 'glyphs2ufo')()
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/__main__.py", line 56, in main
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/glyphslib.py", line 88, in build_masters
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/glyphslib.py", line 79, in write
  File "/usr/local/lib/python2.7/dist-packages/robofab/objects/objectsRF.py", line 471, in save
    glyph = self[glyphName]
  File "/usr/local/lib/python2.7/dist-packages/robofab/objects/objectsBase.py", line 184, in __getitem__
    return self.layers.getDefaultLayer()[glyphName]
AttributeError: 'RFont' object has no attribute 'layers'
```

This comes from the `ufo3k` branch of robofab. In a rather recent issue about this error: https://github.com/robofab-developers/robofab/issues/53 @typesupply states:

> The R* objects in the ufo3k branch is incomplete and is not going to be completed. UFO 3 support is going to be accomplished by a rewrite. The ufo3k branch objects were an experiment that failed.

I was using `ufo3k` because https://github.com/LettError/MutatorMath/blob/master/README.md claims its dependencies as:

> The UFO processing tools in MutatorMath need some additional libraries. Note that anticipating the move to UFO3, MutatorMath already uses the UFO3 branches for Defcon, FontMath and UFOLib. While these modules have received a lot of testing, these are not the main branches yet.
>
>| Library  | Author             | Branch | URL                                              |
| -------- | ------------------ | ------ | ------------------------------------------------ | 
| Defcon   | TypeSupply.com     | ufo3   |  https://github.com/typesupply/defcon            |
| FontMath | TypeSupply.com     | ufo3   |  https://github.com/typesupply/fontMath          |
| Robofab  | RoboFab Consortium | ufo3k  |  https://github.com/robofab-developers/robofab   |

**Comment by jamesgk** [comment]
Based on Tal's comments it would seem that those MutatorMath readme notes are invalid. Personally, I've been using the master branch of RoboFab (and everything else) and that works fine.

**Comment by typesupply** [comment]
In the ufo3k branch, /ufoLib moved away from /robofab into its own package. I'm about 100% certain that the MutatorMath note is referencing the ufoLib package, not the robofab package. Anyway, the UFO 3 branch is an official example implementation of the UFO spec:

https://github.com/unified-font-object/ufoLib

**Comment by graphicore** [comment]
@typesupply indeed I had not installed the ufoLib since it became stand alone, however, it did not solve the problem.

Having installed the master branches of all dependencies I can now successfully export the Glyphs-masters. Exporting instances is failing though:

```
$ glyphs2ufo -g Mirza\ 2\ Masters.glyphs -m masters -n instances
[…]
>>> Writing masters
Traceback (most recent call last):
  File "/usr/local/bin/glyphs2ufo", line 9, in <module>
    load_entry_point('glyphs2ufo==1.0', 'console_scripts', 'glyphs2ufo')()
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/__main__.py", line 59, in main
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/glyphslib.py", line 101, in build_instances
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/interpolation.py", line 43, in interpolate
  File "build/bdist.linux-x86_64/egg/glyphs2ufo/interpolation.py", line 83, in add_masters_to_writer
KeyError: 'com.google.glyphs2ufo.weightValue'
```
However, that's rather another issue.

**Comment by jamesgk** [comment]
The last issue should be fixed as of ddebad2605c7fb2ef1cb9f9d0c1aeccbd799784b. Can we close this?
